src/mesh/parallel_mesh.C

```diff
@@ -290,10 +290,7 @@ dof_id_type ParallelMesh::parallel_max_node_id() const
 
 const Point & ParallelMesh::point (const dof_id_type i) const
```

Comment:
Should `Mesh::point()` actually be `libmesh_deprecated()` in your opinion?  You can catch the return value of `Mesh::node_ref()` as a Point if you really want to...